### ISSUE TITLE
Ubuntu1 user 4 digit Pin

### ISSUE THREAD

**Issue by wmxace** [HelloVolla/ubuntu-touch-beta-tests#123]
I recently got the volla phone with Ubuntu Touch. The first time I started it, it worked fine a d did not ask me for a pin. I left it for a few days. Recharged it and booted it up. A screen showing user Ubuntu1 came up asking me for a pin. I assumed this must be a default pin as I did not set a pin. I have tried various combinations but nothing works. If anyone can help or faced a similar issue please let me know. Don't want to brick it before I get to use it.

**Comment by j2g2rp** [comment]
It is a weird one. I didn't saw it before. Anyway AFAIK you won't break the phone by fail the password. The system will let you to try again but it will make you wait some minutes (more fails will make wait you more minutes) to avoid  a brute force unlock.

If nothing works I would flash it with ubuntu again with the Ubports installer. Be carefull I almost somehow bricked my volla x flashing the device and I had to follow this guide https://rubencarneiro.github.io/rubencarneiro.io//Volla-phone-Repair/ (I don't know what I did wrong but this helped me).
